eo-maven-plugin/src/test/java/org/eolang/maven/ShakeMojoTest.java

```diff
@@ -96,6 +96,10 @@ void getsAlreadyShakenResultsFromCache(@TempDir final Path temp) throws Exceptio
                 .resolve(hash)
                 .resolve("foo/x/main.xmir")
         );
+        Files.setLastModifiedTime(
```

Comment:
@Yanich96 Can we use `save` function instead?

Reply:
@volodya-lombrozo I will create issue